app/services/force/listing_service.rb

```diff
@@ -122,5 +124,33 @@ def self.array_sort!(listing)
       end
       listings
     end
+
+    private_class_method def self.add_cloudfront_urls_for_listing_images(listings)
+      listing_images = ListingImage.all
+
+      listings.each do |listing|
```

Comment:
We can make this `each` block more compact like this:
```ruby
next unless listing['Listing_Images'].present?
listing['Listing_Images'] = set_cloudfront_url(
  listing['Listing_Images'],
  ListingImage.where(salesforce_listing_id: listing['Id'])
)
```

Reply:
addressed